cql3/restrictions/statement_restrictions.hh

```diff
@@ -355,12 +354,10 @@ public:
      */
     const expr::single_column_restrictions_map& get_single_column_partition_key_restrictions() const;
 
-    // Returns any filter that needs to be applied to the partition key. If the partition key restriction is translated
-    // to read_command, it will not be in the filter.
-    //
-    // This is equivalent to pk_restrictions_need_filtering() + get_single_column_partition_key_restrictions().
-    const std::optional<expr::expression>& get_partition_key_filter() const {
-        return _partition_key_filter;
+    // Returns any filter that needs to be applied to a row, but if it fails, ir will fail for all rows in the partition.
```

Comment:
Maybe it will be helpful to add in parentheses that this can be filtering on the partition key or a static column.
Or maybe it can be other things?

Reply:
I'll add the explanation.

Later, if we have an expression e(pk, ck) that cannot be factored, it will be part of the clustering row filter, even though it contains a partition key.